specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/iotSecuritySolutionAnalytics.json

```diff
@@ -0,0 +1,711 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Center",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+    "version": "2019-05-01-preview"
```

Comment:
>2019-05-01-preview [](start = 16, length = 18)

The version doesn't match the directory name. Which one should it be? `2017-08-01-preview` or `2019-05-01-preview`?

Reply:
2017-08-01-preview. fixed